README.md

```diff
@@ -3,12 +3,7 @@
 # OpenHTF
 The open-source hardware testing framework.
 
-[![Build Status](https://travis-ci.org/google/openhtf.svg?branch=master)](https://travis-ci.org/google/openhtf)
```

Comment:
Did I remove non-broken ones? Because at the time, i only removed the broken ones.

Sorry for the long response. I never got a notification.